recording_external_revenue/models/recording_external_revenue_raw.py

```diff
@@ -315,6 +314,16 @@ def _map_recording(self):
         ):
             return self._map_recording_from_catalog_reference()
 
+    def _map_recording_from_product(self, product):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/numigi/odoo-recording-addons/pullRequest?prid=5796004)